### ISSUE TITLE
2022.3 - Input Selects

### ISSUE THREAD

**Issue by danbayliss** [home-assistant-community-themes/midnight#30]
The new version of Home Assistant introduces themeing for input selects. Can you update this theme as currently the input selects are white and stand out against everything else? The new variables are below

```/* input components */
      --input-idle-line-color: rgba(0, 0, 0, 0.42);
      --input-hover-line-color: rgba(0, 0, 0, 0.87);
      --input-disabled-line-color: rgba(0, 0, 0, 0.06);
      --input-outlined-idle-border-color: rgba(0, 0, 0, 0.38);
      --input-outlined-hover-border-color: rgba(0, 0, 0, 0.87);
      --input-outlined-disabled-border-color: rgba(0, 0, 0, 0.06);
      --input-fill-color: rgb(245, 245, 245);
      --input-disabled-fill-color: rgb(250, 250, 250);
      --input-ink-color: rgba(0, 0, 0, 0.87);
      --input-label-ink-color: rgba(0, 0, 0, 0.6);
      --input-disabled-ink-color: rgba(0, 0, 0, 0.37);
      --input-dropdown-icon-color: rgba(0, 0, 0, 0.54);```

**Comment by maartenpaauw** [comment]
Hello @danbayliss. Thanks for addressing the new input select theming options. Feel free to create a pull request.

**Comment by danbayliss** [comment]
Closed with #31 , my first accepted pull request :)

**Comment by maartenpaauw** [comment]
@danbayliss thanks for contributing! Hope many more will come!
